.github/ISSUE_TEMPLATE/package_request.yml

```diff
@@ -0,0 +1,32 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax
+name: Feature request
+description: Request a new package
+title: 'package request: '
+labels: [package-request]
+body:
+  - type: input
+    attributes:
+      label: package link
+      description: A link to the package you'd like to have on the OT-2 on PyPI or condaforge or some other index
+      placeholder: https://pypi.org/project/pandas/
```

Comment:
should we add "eg." or remove `pandas` from these?

Reply:
This is how the [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema) describe it:

> placeholder | A semi-opaque placeholder that renders in the text area when empty. | Optional | String | Empty String |  

Some of their examples have "ex." and some don't.

Reply:
Let's try it without the `ex.` at first and we can change it later if it's not really as good as we think?